src/components/TypeComponents/SimpleSearch/Form/filters/SimpleSearchUUIDFilterField.js

```diff
@@ -29,9 +29,9 @@ const SimpleSearchUUIDFilterField = ({
   const { initialValues, values } = useFormState();
   const { change } = useForm();
 
-  const isSetOrUnset = get(values, `${name}.comparator`) === 'isNull' || get(values, `${name}.comparator`) === 'isNotNull';
-  const relOrAbsValue = get(values, `${name}.relativeOrAbsolute`);
+  const comparatorPresent = isComparatorSpecialCase(values?.[name?.comparator]);
 
+  const relOrAbsValue = get(values, `${name}.relativeOrAbsolute`);
```

Comment:
If we're using optional chaining instead of get, we might as well do that elsewhere in this file as well.